src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java

```diff
@@ -815,10 +816,10 @@ private LayoutBlock buildPlainTextBlock(String text, BlockId blockId) {
         return SectionBlock.builder().text(PlainTextObject.builder().text(text).build()).blockId(blockId.getId()).build();
     }
 
-    private String getStringFromTextFile(String filePath) {
+    private String getStringFromResourceTemplateMailTextFile(String fileName) {
         StringBuilder sb = new StringBuilder();
 
-        try (Stream<String> stream = Files.lines(Paths.get(filePath), StandardCharsets.UTF_8)) {
+        try (Stream<String> stream = Files.lines(Paths.get(getClass().getClassLoader().getResource("templates/mail/" + fileName).getPath()), StandardCharsets.UTF_8)) {
```

Comment:
`getResources(...)` could return `null` if the the resource is not found. Is it necessary to add a null check?

Reply:
It may be good to add in case template names are changed. 

Also, is `/src/main/resources/` not needed in the file path?

Reply:
@benzuzu yup, the method `getClass().getClassLoader().getResource` will locate the file in the resource folder. In the build jar/war file. src/main/resources is no longer available.